src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/DSL.kt

```diff
@@ -319,18 +321,6 @@ interface CourseActivityDsl {
     saturday: Boolean = false,
     sunday: Boolean = false,
   )
-
-  @CourseAllocationDslMarker
-  fun courseAllocation(
-    offenderBooking: OffenderBooking,
-    startDate: String? = "2022-10-31",
-    programStatusCode: String = "ALLOC",
-    endDate: String? = null,
-    payBands: MutableList<OffenderProgramProfilePayBandBuilder> = mutableListOf(),
-    endReasonCode: String? = null,
-    endComment: String? = null,
-    dsl: CourseAllocationDsl.() -> Unit = {},
-  )
```

Comment:
Removed allocations from the course activity DSL for reasons specified above - it caused a circular reference in the builders.